runtime/devnet/src/config/api/mod.rs

```diff
@@ -114,16 +114,16 @@ impl nonfungibles::Config for Runtime {
 
 parameter_types! {
 	pub const MaxXcmQueryTimeoutsPerBlock: u32 = 100;
-	// TODO: What is reasonable.
 
+	// TODO: What is reasonable.
 	pub const IsmpRelayerFee: crate::Balance = crate::UNIT / 2;
 	pub DummyFeeAccount: AccountId = crate::PalletId(*b"dummyacc").into_account_truncating();
-
 }
 
 impl messaging::Config for Runtime {
 	type CallbackExecutor = CallbackExecutor;
-	type FeeAccount = DummyFeeAccount;
+	// Burn fees.
```

Comment:
Assume this is simply a todo, but as we dont have our own token and DOT fees are required for continued operations, we cannnot be burning fees.

Reply:
Devnet seems to be burning transaction payments anyway?

```rust	
type OnChargeTransaction = pallet_transaction_payment::FungibleAdapter<Balances, ()>;
```

Reply:
I will update the devnet to use a proper deal with fees impl to split between treasury and maintenance